{"name": "Проверка неуспешного логина для неполных данных - отсутсвует логин", "status": "passed", "start": 1730235332260, "stop": 1730235332331, "uuid": "7afe7012-e893-4306-9d82-5e89d5480c52", "historyId": "c61dbb6a920d16e51d5811ac4f8f5f7f", "testCaseId": "c61dbb6a920d16e51d5811ac4f8f5f7f", "fullName": "tests.test_login_courier.TestLoginCourier#test_courier_cant_login_without_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "home"}, {"name": "thread", "value": "15252-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}